browser/UHR.js

```diff
@@ -180,7 +183,9 @@ function getStatusObject(xhr) {
 		});
 
 	return {
-		code: xhr.status,
+		// handle IE9 bug:
+		// http://stackoverflow.com/questions/10046972/msie-returns-status-code-of-1223-for-ajax-request
+		code: xhr.status === 1223 ? 204: xhr.status,
```

Comment:
Point in bug description. It exist in IE9 to and fixed only in IE10. It's normal if i place google short link here?